### PR TITLE
Add separate page for Sliceviewer release notes

### LINKED ISSUE #29506: Create a dedicated sliceviewer release notes page
As a developer we want to announce developments to users. The slicevewer has had many changes over this release cycle and as a result it deserves its own sub page of the workbench release notes.

It should be image heavy to show off all of the new features including:

* non-orthogonal view including peak overlays
* roi/line-cut selection and extraction
* general UX improvements

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Adds a separate release notes page for the sliceviewer. It has had a significant number of new features that are worth highlighting on their own.

**To test:**

Build the docs and check they look okay with all of the images included.

Fixes #29506 

*This does not require release notes* because **it is release notes**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by DanielMurphy22 — APPROVED** [review]
Really well done, pictures look great!
Easier to have this merged too before the other core stuff!

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
Actually I should have pointed these out: http://developer.mantidproject.org/Standards/DocumentationGuideForDevs.html#how-to-define-titles-sections-etc
You should be able to batch commit now I've suggested the changes

File: docs/source/release/v5.1.0/sliceviewer.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
+===================
+Sliceviewer Changes
+===================
+
+.. contents:: Table of Contents
+   :local:
+
+New Features
+############
```
[the remark this answers is not in the document]
Comment (martyngigg) [reply docs/source/release/v5.1.0/sliceviewer.rst:9]: Thanks for the reminder of this and I wrote that page :facepalm:

**Comment by martyngigg** [comment]
Thanks @DanielMurphy22. I've rebased and fixed the merge conflicts.

**Review by DanielMurphy22 — APPROVED** [review]
Images look great! Good to go.

**Comment by DavidFair** [comment]
I didn't realise how many new things have gone into slice viewer, it's almost an entire release worth of features in a single area

**Comment by martyngigg** [comment]
Yes I must admit I had to double check the last release too!

**Pull request merged**